ipatests/test_integration/test_cert.py

```diff
@@ -46,3 +47,50 @@ def test_cacert_file_appear_with_option_F(self):
         assert status == "MONITORING"
 
         self.clients[0].run_command(['ls', '-l', '/etc/pki/tls/test.CA'])
+
+
+class TestInstallMasterReplica(IntegrationTest):
```

Comment:
This class name is already taken and sounds like this is an installation test. Could you change it?

Reply:
Will fix it.